### ISSUE TITLE
IOTC_ER_timeout error on my v2 Wyze cams

### ISSUE THREAD

**Issue by joe2k1** [mrlt8/docker-wyze-bridge#628]
I am running an older version - 1.6.4, I noticed my Wyze cam v2 (my v3s are not affected) is not recognized inside the Synology NAS survillenace software. 

I decided to upgrade (to the latest version since many changes have taken place since 1.6.4) Sadly I have not been able to get the new release working. I am using a Synology NAS with docker. The developer and others helped me in a previous setup issue that I posted on [here](https://github.com/mrlt8/docker-wyze-bridge/discussions/440) earlier in the year. I initially had errors getting the MFA working but figured out how to convert the 6 digit 2 factor number into base32. My password does have a special character but when I double it, that doesn’t matter.  I added in a TZ plus the NET MODE is set to LAN. Here is the error that I am getting now:

Invalid credentials? 
400 Client Error: Bad Request for url: https://auth-prod.api.wyze.com/user/login 

Any ideas?

**Comment by mrlt8** [comment]
Not sure what causes this, but I believe some users have issues accessing the wyze api for some reason #505.

You may be able to get around it by using the beta servers with:

```yaml
WYZE_BETA_API=True 
```
and/or use the android key:

```yaml
WYZE_APP_API_KEY=RckMFKbsds5p6QY3COEXc2ABwNTYY0q18ziEiSEm
```

**Comment by joe2k1** [comment]
Thank you for the response. I was able to resolve this issue. It involved a significant amount of trial and error. In the end I was remove the wyze_user and wyze_password, restarting. I added those two variables back in and removed the two suggested variables you recommended. I was to successfully login without that invalid credentials+400 client error appearing. I am relieved to be on the latest build now. I will close this thread now and leave it up as a troubleshooting document for others who may experience this strange intermittent issue.
